db/schema.rb

```diff
@@ -112,13 +122,14 @@
     t.bigint "preferred_language_id", null: false
     t.datetime "start_at", null: false
     t.integer "expected_duration", null: false
-    t.integer "number_of_children", null: false
+    t.integer "number_of_children"
     t.bigint "notified_user_ids", default: [], array: true
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.bigint "race_id"
     t.text "notes"
     t.bigint "unavailable_user_ids", default: [], array: true
+    t.boolean "preferred_language_override", default: false
```

Comment:
Can we ensure that this is the only line that shows changed in this file (in addition to the schema version above)